src/plugins/intel_gna/gna_graph_compiler.cpp

```diff
@@ -882,6 +882,18 @@ void GNAGraphCompiler::PoolingPrimitive(InferenceEngine::CNNLayerPtr layer) {
     uint32_t num_padding = ALIGN(hw_in, 8) - hw_in;
     size_t num_data_bytes_in = c_dim_in * (hw_in + num_padding) * inputs->getPrecision().size();
 
+    if (dnn->new_num_conv_columns) {
```

Comment:
this code seems to be the same as for PWL primitive, can it be shared with a separate function?

Reply:
The same only this code:
if (dnn->new_num_conv_columns % num_columns == 0) {
num_rows = dnn->new_num_conv_columns / num_columns;
} else {
num_columns = dnn->new_num_conv_columns;
}
I think it's unnecessary to create a function for a four lines code.